### PR TITLE
Created Issue Triage file

### PR DESCRIPTION
**Pull request by kannankvs**
This PR has been raised as the original [PR#135](https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/135). And has been closed due deletion of head repository.

<strong>KVSK</strong> the below comments were updated as suggested. The documents created are from the reference given by Ansible team, hence the phrases are taken reference from the Ansible team. 

<strong>stalabi1</strong> Line :  66 : change 'doesn't' to 'don't'
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 125 : How about saying something along the lines of 'In order to gain comfort with prioritizing issues, consulting with experienced project members on issues is highly encouraged.'
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 13 : How about saying:
'The main goals of an issue triage are to categorize open issues based on priority and to ensure that these issues contain adequate information for contributors to work on them.'
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line :  19 - 21 :  Intent and purpose are synonyms. How about saying :
"A triage helps resolve issues quickly by:
Ensuring the intent of an issue is clear. The issue should clearly describe the problem that the end user is experiencing as well as the steps to reproduce the problem."
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line :  23 :  I think you can omit line 23 and say:
- Streamlining the development process by identifying and closing duplicate issues.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 34 : When creating a new issue, the author must provide all of the information required by the project's GitHub issue template.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 42 : A bug report should contain explanation of what the problem is, what the expected outcome is, and how to reproduce the problem. Additionally, any other supporting information such as screenshots, console logs, and environment details should be included in the bug report. 
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 63 : To make issue handling easier for everyone, it is suggested to:
<strong>jeff-yin</strong> Line : 63 : We also shouldn't use the phrase "rule of thumb" according to Inclusive Language guidelines at Dell.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 81 : If you receive a notification that additional information was provided but you are no longer working on issue triage, then you should delegate the issue to the current person working on issue triage.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 87 : Make sure it's not a duplicate by searching existing issues using related terms from the issue title and description. If you think the issue may be a potential duplicate and can't find the original existing issue, then please reach out to one of the maintainers and ask for help. If you identify that the issue is a duplicate of an existing issue:
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 99 : 2. If maintainers aren't able to address the issue in a timely manner, then label the issue with `help wanted` and an optional `beginner friendly`. Also, include pointers to the problematic source code. Doing this allows us to solicit help from the community to get the issue resolved.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 69 : -  If an issue is unclear, then try to edit the title and description for more clarity or leave a comment requesting that edits to the issue be made.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 77 : If the author provides the standard information but you are still unable to triage the issue, then request additional information. Do this kindly and politely because you are asking for more of the author's time. Label issue with `triage/needs-information`.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 104 :  1. Either request that [more information](#2-ensure-the-issue-contains-basic-information) is needed to investigate the issue more thoroughly. Provide details in a comment.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 105 : 2. Either [delegate further investigations](#investigation-of-issues) to someone else. Provide details in a comment.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 109 : 1. Kindly and politely add a comment briefly explaining why the behavior is believed to work as intended and then close the issue.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 118 : 2. Otherwise, move on to [prioritizing the issue](#4-prioritization-of-issues). Assign the appropriate priority label to the issue, add the appropriate comments to the issue, and remove the `needs-triage` label.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 121 : In general, bugs and feature request issues should be labeled with a priority.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 123 : This can be the most difficult task when triaging issues since it requires a lot of knowledge, context, and experience before being able to start feeling comfortable adding a certain priority label.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 65 : - Make sure that issue titles are named to explain the subject of the issue, have correct spelling, and don't include irrelevant information and/or sensitive information.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 131 : | `priority/critical`               | Highest priority. Must be actively worked on as someone's immediate top priority. 
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 133 : | `priority/low`                    | Lowest priority. Possibly useful, but not critical to functionality.  
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 137 : 1. If an issue has been categorized and any of the following criteria apply to it, then the issue should be labeled as critical and must be actively worked on as someone's immediate top priority.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 142 :   - Multiple users experience a severe problem affecting their business, users, etc.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 146 :   4. Assign the issue or ask a maintainer for help assigning the issue to someone as their immediate top priority.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 158 :  1. If the issue is deemed possibly useful but is a low priority, then label the issue `priority/low`.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 159 :  2. The functional impact of an issue determines how high of a priority the issue is.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 168 : 1. Kindly and politely add a comment which will notify the users subscribed to the issue of any updates.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 178 : Even if you don't have the time or the knowledge to investigate an issue we highly recommend that you upvote the issue if you happen to have the same problem. If you have further details that may help with investigating the issue please provide as much information as possible.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 182 : Part of issue triage should also be triaging of external PRs. The main goal should be to make sure PRs from external contributors have an owner/reviewer and are not forgotten.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 169 :    - Explain that the issue would be nice to get resolved, but it isn't prioritized to be worked on by maintainers for the foreseeable future.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 170 :    - If possible or applicable, try to help contributors get started by adding pointers to relevant source code or files as well as adding any helpful input on how to resolve the issue.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 164 :  Due to constraints, maintainers aren't always able to resolve issues in a timely manner, especially those of lower priority. For such issues, consider requesting help from the community. Use your best judgement. In general, requesting help from the community means that a contribution has a good chance of getting accepted and merged.
<strong>KVSK</strong> addressed 

<strong>stalabi1</strong> Line : 166 : In many cases, the issue author or the community is most suitable to contribute changes since they're experts in their domain. It's also quite common that someone has tried to get something to work using the documentation without success and made an effort to get it to work and/or reached out to the community to get the missing information.
<strong>KVSK</strong> addressed

### CODE REVIEW

File: docs/ISSUE_TRIAGE.md
Comment (stalabi1) [inline docs/ISSUE_TRIAGE.md:125]:
```suggestion
In order to gain comfort with prioritizing issues, consulting with experienced project members on issues is highly encouraged.
```

File: docs/ISSUE_TRIAGE.md
Comment (stalabi1) [inline docs/ISSUE_TRIAGE.md:166]:
```suggestion
In many cases, the issue author or the community is most suitable to contribute changes since they're experts in their domain. Also, it's very common for someone to seek help from the community on a problem.
```

File: docs/ISSUE_TRIAGE.md
Comment (stalabi1) [inline docs/ISSUE_TRIAGE.md:23]:
```suggestion

```

**Comment by stalabi1** [comment]
I clarified some comments. Once those are addressed, I'll approve.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Addressed as suggested

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Addressed as suggested

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Addressed as suggested

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: removed as suggested

**Review by stalabi1 — APPROVED** [review]

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:102]: This isn't grammatical and is a little hard to read. (Having two successive "Either" statements is confusing.)

Please delete the two "Either"s at the beginning of these items and do one of the following:

- Precede the two items with "Either", then add an "or" between the two items.
- Precede these two list items with "Do one of the following:" (This implies the Either/or.)

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:175]: This wording is unclear. I will try to provide an alternate phrasing, but am somewhat guessing at what was intended. I think that if we ask people to label an issue "needs information" because it is difficult to reproduce, even though they do think it has enough information, we should explicitly ask them to explain why they are labeling it as "needs information". 

```
When an issue has all basic information provided, but the reported problem cannot be reproduced at a first glance, label the issue `triage/needs-information` and post a comment explaining why this label is being applied.
```

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:186]: Delete the word "does".

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:188]: Was there intended to be more content here? The "ask a question" and "new feature" types aren't shown, and no actual workflow steps are described.

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
I have posted a few minor follow-up comments/requests and a question for the contents of this document.

Other than these items, this document looks fine.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:188]: @kerry-meyer  :  As of now there is no additional content. If you have any suggestion for adding more content related to "Ask a Question" and/or "New Feature" types, kindly provide the content. 
Alternatively, you can approve this with the current content if that is acceptable.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:102]: Addressed as suggested

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Addressed as suggested

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md]: Thanks for correcting this.

Resolved.

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md:102]: There are still two preceding "Either"s. Please delete the one for alternative "2".

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:113]: Please add some description here or at least add a comment that indicates an action item for maintainers to provide the additional information. There is already a suitable matching action described for feature requests (prioritization and removal of the 'needs-triage' label). That could be copied here to complete this section.

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md:190]: Please at least provide the proposed additional steps of the workflow expected for handling of a "Bug" type issue (leading up to and including closing of the issue) and/or add a comment indicating that this is a placeholder for future content and stating that maintainers will provide additional details in the future. 

Alternatively, this section can be deleted since it is mostly already covered by the previous content of the document. Since the full workflow is beyond the scope of "triage", it could, and probably should be included in one of the other documents ("maintainer's guide" or "contributor's guide") if it's not already there and referenced here instead of providing a partial section.

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
Thanks for posting responses to my comments.

I have posted follow-on comments, including one new request for additional content in the "Bug reports" section that I noticed when checking the context for my follow-on response on the "workflow" section.

File: docs/ISSUE_TRIAGE.md
Comment (jeff-yin) [reply docs/ISSUE_TRIAGE.md:190]: I don't think we need this. Let's just delete it. 
If it's strictly required for OneAnsible, we can use the same content that other components have been using. Otherwise let's just get rid of it; it's extraneous detail.

Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:190]: @jeff-yin This requirement is coming from OneAnsible. I am not clear on what you mean by content from "other components". Which other components are you referring to?

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:113]: We will discuss with server storage team and get back to you.

**Comment by kannankvs** [comment]
wrt to comment for line number 188, we will discuss with server storage team and will get back to you.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:190]: @jeff-yin : This document is the guide for "core" maintainer/developer/lead who will do the first round of triaging. Do you mean to remove this document completely? Or, do you mean to remove this particular point?

Comment (jeff-yin) [reply docs/ISSUE_TRIAGE.md:190]: "other components" -- just meaning Storage and Servers.

I just mean to remove this particular point, not the entire section. I wrote this in reply to this thread about the "GitHub issue management workflow" section, so I'm not sure why it was interpreted that I meant to remove the document completely.

File: docs/ISSUE_TRIAGE.md
Comment (jeff-yin) [inline docs/ISSUE_TRIAGE.md:192]: "Github" is misspelled here.

File: docs/ISSUE_TRIAGE.md
Comment (jeff-yin) [inline docs/ISSUE_TRIAGE.md:70]: I don't think this line is needed.

File: docs/ISSUE_TRIAGE.md
Comment (jeff-yin) [reply docs/ISSUE_TRIAGE.md:113]: If something is both "not a bug" but also "does not work as intended/by design" then it seems appropriate to convert/tag the issue into a Feature Request. Perhaps you can add language suggesting this.

Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:113]: @jeff-yin we will discuss with server storage team and will get back to you.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:70]: Okay. We will discuss with server storage team and will get back to you.

Comment (jeff-yin) [reply docs/ISSUE_TRIAGE.md:70]: I just think that saying "rules are applicable to both new and existing issues" is already implied (why would they not be applicable?) and doesn't really add any value to the document.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Server/storage team is planning to create a workflow diagram and add to this document later. Until then, the section "##GitHub issue management workflow" is not required and hence removed. This document will be enhanced in future version when the diagram is available.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Yes, this comment was discussed with server/storage team. It is agreed that "It does not work as intended/by design" is same as "It's a bug/it can be reproduced". Hence the additional category/line "It does not work as intended/by design" has been removed.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: This section has been removed as per above discussion thread.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Removed as suggested.

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:187]: Please delete this section. It seems redundant and doesn't fit well with the organization of the document.

The points described here can be incorporated into the "Bug Reports" section if desired.

Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:190]: This section has been updated with flowchart as suggested by server storage team. Hence request to kindly review

File: docs/ISSUE_TRIAGE.md
Comment (jeff-yin) [inline docs/ISSUE_TRIAGE.md:200]: This should just be a PNG that is attached in the doc. Not sure why it needs to be ASCII art. But it's ok for now.

**Review by jeff-yin — APPROVED** [review]

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:214]: typo/spelling: Change "Dupicate" to "Duplicate".

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:285]: Change "contribution" to "request" in "Immediately close the contribution ..."

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md:186]: Because real content has been provided, I am marking this issue "Resolved".

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md:190]: I am marking this issue "Resolved" because the recent modifications make it no longer relevant.

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:246]: Shouldn't there be a workflow title line here, similar to the one for "Bug" type issues?

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
I am resolving some issues and opening new ones based on the most recent updates.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Updated as suggested

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: Updated as suggested

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md]: addressed

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [inline docs/ISSUE_TRIAGE.md:284]: Please also fix the other instances where the word "contributions"  appears instead of "request".  There's one in this line and also instances on lines 286 and 288.

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md]: There was another instance of the same thing in the referenced line. Please make the same change for that instance and other similar instances in other lines. See the new comment for this below specifying the current line numbers that need this change.

File: docs/ISSUE_TRIAGE.md
Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md:200]: I'm resolving this conversation because Jeff has approved the PR.

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
The requested additional heading has been added, but it is not visible in the displayed document because it is enclosed in documentation markers.

Please remove the documentation markers to make the heading visible to the reader.

File: docs/ISSUE_TRIAGE.md
Comment (kannankvs) [reply docs/ISSUE_TRIAGE.md:246]: We are able to see the headings separately. The content for the first flowchart starts in line : 196 followed by document marker at line : 200 to mark the start of first flowchart. Then the document marker ends at line : 246 for the completion of first flow chart. After which the heading for second flow chart is given at line : 248, then the document marker starts at line : 250 to mark the start of second flowchart. The document marker ends at line : 281 for the completion of second flow chart. Would request to verify once in this [md file format](https://github.com/ansible-collections/dellemc.enterprise_sonic/blob/ed462e20322506c2050f6042718dbbc7b4615d7a/docs/ISSUE_TRIAGE.md).

Comment (kerry-meyer) [reply docs/ISSUE_TRIAGE.md:246]: Okay. I don't know why I thought the second heading wasn't being displayed. I see that it is.

I will resolve this last issue, approve the PR, and merge it.

**Review by kerry-meyer — APPROVED** [review]

**Pull request merged**
